packages/panels/src/components/sidebar-menu/SidebarMenu.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Column } from "@stenajs-webui/core";
+import * as React from "react";
+import {
+  SidebarMenu,
+  SidebarMenuContainer,
+  SidebarCloseButton,
+} from "@stenajs-webui/panels";
+import {
+  SidebarMenuHeading,
+  SidebarMenuLink,
+  SidebarMenuCollapsible,
+} from "@stenajs-webui/panels";
+import {
+  faBook,
+  faChartBar,
+  faPaperPlane,
+  faUserFriends,
+} from "@fortawesome/free-solid-svg-icons";
+import { Icon } from "@stenajs-webui/elements";
+
+export default {
+  title: "panels/SidebarMenu",
+};
+
+export const Overview = () => {
```

Comment:
Add stories with all permutations of props. Well, more at least :)
For example, with and without iconLeft on all components that have it.